rpms/group/admin-tools.pan

```diff
@@ -0,0 +1,25 @@
+# Based on SL5.9 group definition
+
+unique template rpms/group/admin-tools;
```

Comment:
Note that `ncm-spma` is now supporting YUM groups directly, without expanding them in the template. This may be a more maintainable solution. Look at `el7.x` `rpms/base.pan` for an example. You may want to keep the `rpms/group` template for easier backward compatibility but use the YUM group inside it.